.github/workflows/build.yml

```diff
@@ -48,12 +48,18 @@ jobs:
         name: bzip3-${{ github.sha }}
     - name: Extract source package
       run: tar --strip-components=1 -xf bzip3-${{ github.sha}}.tar.gz
+    - name: Fetch examples
+      run: |
+        mkdir examples
+        cd examples
+        wget https://github.com/kspalaiologos/bzip3/raw/${{ github.sha }}/examples/shakespeare.txt \
+             https://github.com/kspalaiologos/bzip3/raw/${{ github.sha }}/examples/shakespeare.txt.bz3
```

Comment:
As in the other two places, I'm trying to download the two files directly.
I wanted to avoid adding them to the package in the Makefile.am dist task, which would probably inflate the release package.
:black_cat: I wish there was another, better way, but it doesn't seem possible for me to do it.